src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/dto/LocationGroupDtoListTest.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.locationsinsideprison.dto
+
+import com.fasterxml.jackson.annotation.JsonInclude
+
+@JsonInclude(JsonInclude.Include.ALWAYS)
```

Comment:
This class should be a data class and also included in the `LocationPrisonIdResourceTest.kt` file instead